corehq/apps/app_manager/suite_xml/case_tile_templates/clinic_and_unit_tile_with_centered_last_field.xml

```diff
@@ -8,7 +8,7 @@
   <field>
```

Comment:
this commit is best viewed with [`?w=1`](https://github.com/dimagi/commcare-hq/pull/33128/commits/c9a9288ee85f32ef2bae9a80d6903f37d7af71da?w=1)